Dockerfiles/Dockerfile.cvp

```diff
@@ -0,0 +1,16 @@
+FROM registry.fedoraproject.org/fedora:33
```

Comment:
The name of dockerfile is really confusing. It has nothing to do with CVP.
It should be just about CI. Moreover I do not see an reason why we should mix files for midstream and CI in the same directory. Files for midstream should be just in its own directory.

And files for CI in different one. And ideally name of directory should be self explanatory. The current name "Dockerfiles" does nto say much to me.

Reply:
Dockerfiles is a convention I followed from other projects on Github. 
We can definitely segregate the files to make them more comprehendible.